src/lib/index.ts

```diff
@@ -0,0 +1,277 @@
+import algoliasearchHelper from 'algoliasearch-helper';
```

Comment:
Why isn't `src/lib/index.ts` in `src/widgets/index/`?

Reply:
this is also confusing since the name is "index" and it's not an index file

Reply:
The widgets are the binding between the connector + the Vanilla renderer. It's not the case for this one which is renderer agnostic. That's why it's inside `lib` rather than `widgets` it's part of the "core" of InstantSearch.

Reply:
It think that the reasoning behind @samouss' decision is that when we'll create several InstantSearch packages (e.g. `instantsearch-core`, `instantsearch-vanilla`, etc.), `index` will be exported from `instantsearch-core` because it doesn't include anything render. Thus it makes sense not to consider it as a widget but still to export it in `instantsearch-vanilla`, which right now is the `widgets/` folder.

Is that right @samouss?

Reply:
Yes it is.

Reply:
Well, we also have some widgets in `widgets` that doesn't include anything related to rendering (ex: `analytics` or `configure`). For me is is more about usage than about what is rendered or not.

Reply:
The widget `analytics` is tied to the renderer since we rely on the DOM e.g `location`. This is true for `configure` though. Anyway, we won't debate for ages about such small details. I'm fine with a path change, it doesn't matter.

Reply:
https://github.com/algolia/instantsearch.js/pull/3892/commits/a8e6d3df4856cb1021f9539f3eaef4176eb77317